Firestore/core/src/util/exception.h

```diff
@@ -37,6 +37,7 @@
 
 #include "Firestore/core/src/util/string_format.h"
 #include "absl/base/attributes.h"
+#include "absl/base/config.h"
```

Comment:
I noticed that the header was missing -- `ABSL_HAVE_EXCEPTIONS` is defined in it.